cli/src/app.rs

```diff
@@ -347,6 +347,54 @@ pub fn add_subcommands(command: Command) -> Command {
         )
         .subcommand(extensions::command());
 
+    #[cfg(unix)]
+    {
+        app = app.subcommand(
+            Command::new("sandbox").hide(true).about("Run an application in a sandbox").args(&[
+                Arg::new("allow-read")
+                    .help("Add filesystem read sandbox exception")
+                    .long("allow-read")
+                    .value_name("PATH")
+                    .value_hint(ValueHint::FilePath)
+                    .action(ArgAction::Append),
+                Arg::new("allow-write")
+                    .help("Add filesystem write sandbox exception")
+                    .long("allow-write")
+                    .value_name("PATH")
+                    .value_hint(ValueHint::FilePath)
+                    .action(ArgAction::Append),
+                Arg::new("allow-run")
+                    .help("Add filesystem execute sandbox exception")
+                    .long("allow-run")
+                    .value_name("PATH")
+                    .value_hint(ValueHint::FilePath)
+                    .action(ArgAction::Append),
+                Arg::new("allow-env")
+                    .help("Add environment variable access sandbox exception")
+                    .long("allow-env")
+                    .value_name("ENV_VAR")
+                    .num_args(0..=1)
```

Comment:
This seems like it might cause some parsing issues since we accept positional arguments.

```
phylum sandbox --allow-env cat ./dangerous_script.sh
```

Due to the nature of this command, there will always be arguments after `--allow-env`. So when will clap use the 0 arg version? Only when the following argument starts with `--`?